src/files.js

```diff
@@ -884,6 +887,34 @@ FOAM_FILES([
   { name: "foam/nanos/pool/AbstractFixedThreadPool" },
   { name: "foam/nanos/pool/ThreadPoolAgency" },
 
+  // md
```

Comment:
Later on I would like to move these to md.js, so we don't increase the size of the regular app. Like we do for nanos.js.